ui/pages/smi_results.js

```diff
@@ -11,27 +11,25 @@ const styles = { paper : { maxWidth : '90%',
   margin : 'auto',
   overflow : 'hidden', } };
 
-class SMIResults extends React.Component {
-  componentDidMount () {
+const SMIResults = ({ updatepagepath, classes }) => {
+  useEffect(() => {
     console.log(`path: ${getPath()}`);
```

Comment:
@Rajdip019 What's the reason to console log values project-wide?

Reply:
We can just simply remove this.